### DISCUSSION TITLE
Please help with graph

### DISCUSSION THREAD

**Discussion by mucki12** [matt8707/ha-fusion#393] [Q&A]
First of all, thank you very much for this dashboard! Will be a gamechanger in my opinion as soon as the main frame gets the same possibilities as the sidebar!

A question (or a minor bug?) about graph in sideboard: shouldn't the graph represent the line of the day?

Home Assistant:
![image](https://github.com/matt8707/ha-fusion/assets/92673795/510a9e46-f7f4-42a7-bb3f-ee4f8af57869)

HA Fusion configuration (same entity):
![image](https://github.com/matt8707/ha-fusion/assets/92673795/71c6f69e-51fd-4070-a9a4-663dfdefe198)

HA Fusion result:
![image](https://github.com/matt8707/ha-fusion/assets/92673795/1897a60f-f341-474f-a012-b1a14b340687)

When I move the mouse over the graph, several days were displayed...

In my opinion the code looks good  
```
- id: 7696038957693
    type: graph
    entity_id: sensor.leistung_pv_pt2_watt_ohne_nachkomma
    name: Erzeugung
    stroke: null
    period: day
```

**Comment by knudsvik** [comment]
It seems like the period is actually data frequency / aggregation, not the time period of the x-axis.
